source/base/data_out_base.cc

```diff
@@ -850,11 +850,6 @@ namespace
      * technique.
      */
     std::ostream &stream;
-
-    /**
-     * The flags controlling the output
-     */
-    const DataOutBase::GmvFlags flags;
   };
```

Comment:
Unfortunately this raises a warning (see http://cdash.kyomu.43-1.org/viewBuildError.php?type=1&buildid=8444).